processors/minsev/example_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package minsev // import "go.opentelemetry.io/contrib/processors/minsev"
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+
+	api "go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/sdk/log"
+)
+
+const key = "OTEL_LOG_LEVEL"
```

Comment:
`OTEL_LOG_LEVEL` is a level of the SDK's internal logger.
It is not well defined in the specification, but this is how (AFAIK) everyone is using it.